velox/type/Type.h

```diff
@@ -1448,6 +1553,13 @@ struct CppToType<Timestamp> : public CppToTypeBase<TypeKind::TIMESTAMP> {};
 template <>
 struct CppToType<Date> : public CppToTypeBase<TypeKind::DATE> {};
 
+template <>
+struct CppToType<ShortDecimal> : public CppToTypeBase<TypeKind::SHORT_DECIMAL> {
+  static auto create() {
+    return SHORT_DECIMAL(18, 0);
```

Comment:
This doesn't look right. @majetideepak @karteekmurthys Would you double check?

Reply:
@karteekmurthys did you miss removing this along with the `LONG_DECIMAL`?